source/frontend/src/features/leases/detail/LeasePages/tenant/PrimaryContactWarningModal.test.tsx

```diff
@@ -69,9 +69,9 @@ describe('PrimaryContactWarningModal component', () => {
         ...getMockApiLease(),
         tenants: [
           {
-            ...getMockApiLease().tenants[0],
-            primaryContactId: undefined,
-            primaryContact: undefined,
+            ...getMockApiLease().tenants![0],
```

Comment:
is the ! really necessary?

Reply:
yes. there is a problem with the current version of typescript that we are using. once we move to a newer version these ! wont be necessary